src/components/PreviewComment/index.js

```diff
@@ -103,7 +103,7 @@ export const styles = StyleSheet.create({
   lineBeforeProfile: {
     height: 9,
     borderLeftWidth: 1,
-    borderLeftColor: COLORS.balance_gray,
+    borderLeftColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border color has been changed from `COLORS.balance_gray` to `COLORS.gray200`. Make sure this color change is consistent with the rest of your application's design system and doesn't negatively impact the user experience.
```diff
- borderLeftColor: COLORS.balance_gray,
+ borderLeftColor: COLORS.gray200,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->